### PR TITLE
Added MGE Comet detection

### PR DESCRIPTION
**Pull request by kate66**
This is to add OS detection of MGE Comit type UPSes.

Additionally changed includes/discovery/sensors/current/mgeups.inc.php to check the number of active phases from the MIB output as another MIB that appears to function differently between device types. The new detection shouldn't negatively affect current deployments as it's simply counting the number of MIBs in the tree that it's actively pulling information from.

- [ Y ] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [ Y ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

File: includes/discovery/sensors/current/mgeups.inc.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // MGE UPS
 if ($device['os'] == 'mgeups') {
     echo 'MGE ';
-    $oids = trim(snmp_walk($device, '1.3.6.1.4.1.705.1.7.1', '-OsqnU'));
+    $oids = trim(snmp_walk($device, '1.3.6.1.4.1.705.1.7.2.1.5', '-OsqnU'));
```
Comment (murrant) [inline includes/discovery/sensors/current/mgeups.inc.php]: Why did you change the OIDs?  Do the new OIDs support other devices such as Galaxy and Evolution?

Comment (kate66) [inline includes/discovery/sensors/current/mgeups.inc.php]: The OID change is because on Galaxy/Evolution 1.3.6.1.4.1.705.1.7.1.1/2/3/etc are the number of phases. On the Comit 1.3.6.1.4.1.705.1.7.1 equal to 3 (the number of phases). On all three 1.3.6.1.4.1.705.1.7.2.1.5.1/2/3/etc is equal to the value current of that phase. If only 1.3.6.1.4.1.705.1.7.2.1.5.1 exists, then it's a single phase system.

Comment (kate66) [inline includes/discovery/sensors/current/mgeups.inc.php]: And to clarify, the OIDs that I used to count the number of phases are used about 4 lines down to gather the data to add to the database.

**Comment by murrant** [comment]
Thanks for the explanation.  I've been putting the oid names in a comment whenever I use a numeric oid, I think it helps a lot with readability.

Can you add that?  Then we are probably good to merge.

**Comment by kate66** [comment]
Those are the actual values by the way.

.1.3.6.1.2.1.1.1.0 = STRING: "Comet "
.1.3.6.1.2.1.1.2.0 = OID: .1.3.6.1.4.1.705.1.2

**Comment by laf** [comment]
image
